tests/smoke.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -e
+
+set -o pipefail
+
+TESTS="pwd, lsb_release -a, uname -a, ip a, lscpu, vmstat, lsblk, false"
+LOGS="/tmp/TESTS_$(date +%s)_"
+
+[ -z $TEST_DIR ] && (echo "ERROR: TEST_DIR not defined"; exit 1)
+[ -d $TEST_DIR ] || (echo "ERROR: no TEST_DIR"; exit 1)
+
+run() {
+    # shellcheck disable=SC2039
+    local test="$1"
+    test_case_id="$(echo "${test}" | awk '{print $1}')"
+	result_dir="${TEST_DIR}/$(date +%s.%N)-${test_case_id}"
+	mkdir ${result_dir}
+    echo
+    echo "Running ${test_case_id} test..."
+    local exit_code=0
+    eval "${test}" || exit_code="$?"
+    echo "Exit code: $exit_code"
+
+    if [ "${exit_code}" -ne 0 ]; then
+		touch ${result_dir}/failed
+    fi
+    return "${exit_code}"
+}
+
+failed=0
+while [ -n "${TESTS}" ]; do
+    test_cmd="$(echo "${TESTS}" | awk -F',' '{print $1}')"
```

Comment:
there's an easier way to iterate over the tests
~~~
IFS=","
for test_cmd in $TESTS ; do
~~~

Reply:
it was copy paste from test-definitions. I'll fix it.